PTIR part1

qiuting1995 opened the pull request:
I learned the part3.1 in the paper, which is about Image local Cost. It's a very important part for intelligent scissors because it decide how we define the edge of the image.
The next step for me is to learn part3.2, which is the algorithm 'Live-Wire 2-D DP graph search' and try to realize it in code.

qiuting1995 committed:
README_part3.1

qiuting1995 committed:
Delete essai.txt

qiuting1995 committed:
Delete testqiu.txt

qiuting1995 committed:
part1

What I understand in part3.1(Image Local Cost).

troussil commented:
Ok, but before merging, I want you to put your document into a specific directory (eg. scissors/doc)

qiuting1995 committed:
Create readme

qiuting1995 committed:
Delete readme

qiuting1995 committed:
README_part3.1

qiuting1995 committed:
part1

What I understand in part3.1(Image Local Cost).

qiuting1995 committed:
Delete PTIR_part1.docx

qiuting1995 committed:
Delete README.md

qiuting1995 commented:
Morning,
Thank you for your letter. I have put them into a directory now:)


Yours,
QIU TING

2017-03-27 9:45 GMT+02:00 troussil <notifications@github.com>:

> Ok, but before merging, I want you to put your document into a specific
> directory (eg. scissors/doc)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/troussil/segmentation/pull/8#issuecomment-289378124>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AY4Udd7WPZ0ooXEFnagkJ6D-JnjOTMoAks5rp2khgaJpZM4MoyvT>
> .
>

troussil commented on README.md:
```diff
@@ -1,2 +0,0 @@
-# segmentation
-Implémentation de plusieurs algorithmes de segmentation d'images
```
sorry, but you should not change this file. It describes the whole segmentation project (and you're not alone on this project). Please, revert your changes.

qiuting1995 committed:
part3.1 in paper

In this part, I did some reseached in the flag of the image edge(3.1 Local Costs) in the paper. In the file 'PTIR_part1', I write how I understand of this part and take some notes.

qiuting1995 committed:
Delete PTIR_part1.docx

qiuting1995 committed:
Delete README.md

qiuting1995 committed:
Create a

qiuting1995 committed:
part3.1 in paper

In this part, I did some reseached in the flag of the image edge(3.1 Local Costs) in the paper. In the file 'PTIR_part1', I write how I understand of this part and take some notes.

qiuting1995 committed:
Delete a

qiuting1995 committed:
Initial commit

qiuting1995 commented:
I have changed the README back to the initial one.
